fendermint/actors/f3-light-client/src/types.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021-2024 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+//! Type definitions for the F3 Light Client actor.
+//!
+//! This module defines the core types used by the F3 Light Client actor,
+//! including the light client state structure that tracks F3 finality
+//! from the parent chain.
+
+use fvm_ipld_encoding::tuple::{Deserialize_tuple, Serialize_tuple};
+use fvm_shared::clock::ChainEpoch;
+
+/// F3 Light Client State - maintains verifiable parent finality from the parent chain.
+///
+/// This structure represents the essential state needed to track F3 finality:
+/// - Instance ID: The current F3 instance (can increment during protocol upgrades)
+/// - Finalized Epochs: Complete chain of finalized epochs (not just the latest)
+/// - Power Table: Current validator power table (can change between instances)
+///
+/// This state is extracted from F3 certificates received from the parent chain
+/// and stored by the actor for use in finality proofs.
+#[derive(Deserialize_tuple, Serialize_tuple, Debug, Clone, PartialEq, Eq)]
+pub struct LightClientState {
+    /// Current F3 instance ID
+    pub instance_id: u64,
+    /// Finalized chain - full list of finalized epochs
+    /// Matches ECChain from F3 certificates
+    /// Empty initially at genesis until first update
+    pub finalized_epochs: Vec<ChainEpoch>,
```

Comment:
What's the point of having it here? We can't use it to initialize `F3Client`, because the latter would need the finalized sequence of tipsets (which is optional anyway), not just epoch numbers.